src/main/java/org/sagebionetworks/web/server/servlet/SynapseClientImpl.java

```diff
@@ -3127,4 +3128,16 @@ public ColumnModelPage getPossibleColumnModelsForViewScope(ViewScope scope, Stri
 	public Boolean isUserAllowedToRenderHTML(String userId) throws RestServiceException {
 		return getHtmlTeamMembers().contains(userId);
 	}
+	
+	@Override
+	public ProjectDisplayBundle getCountsForTabs(Entity entity) throws RestServiceException {
+		
+//		try {
+			//org.sagebionetworks.client.SynapseClient synapseClient = createSynapseClient();
+			return new ProjectDisplayBundle(false, true, false, false, false, false);
```

Comment:
Each tab will require a different service call, until the PLFM feature is added.
For example:
getV2WikiPage for wiki, getChallengeForProject() || ChallengeClientImpl.getSharableEvaluations() for challenges.

Logic for determining if ChallengeTab should be visible has been moved to the EntityPageTop, so tab.setTabListItemVisible(true); must be removed from ChallengeTab